pkgs/top-level/aliases.nix

```diff
@@ -813,7 +813,10 @@ mapAliases ({
   postgis = postgresqlPackages.postgis;
   timescaledb = postgresqlPackages.timescaledb;
   tsearch_extras = postgresqlPackages.tsearch_extras;
-
+  pinentry =
+    pinentryFlavors.all // pinentryFlavors
+    // { flavors = lib.attrNames pinentryFlavors; }; # added 2024-01-15
+    # alternatively: throw "'pinentry' has been removed. Pick an approprivate version from 'pinentryFlavors'";
```

Comment:
We should list them statically to not modify them in the future and just make them work with the conversion script